templates/user/dashboard/navbar.tmpl

```diff
@@ -75,30 +75,28 @@
 		{{end}}
 
 	{{if .ContextUser.IsOrganization}}
-		<div class="right stackable menu">
```

Comment:
Actually I think this needs to be `<div class="menu">` as without it you don't get the color highlight to say which view is being used.

Reply:
What about changing to `<div class="org-menu">`? And then change the CSS according to that class?